### DISCUSSION TITLE
Dicas, livros e cursos de especialização

### DISCUSSION THREAD

**Discussion by xJmsx** [frontendbr/forum#1243] [Dicas]
E aí pessoal, bom dia!

Há quase um ano eu resolvi ingressar e tive meu primeiro contato com algo da área (sempre tive o sonho de trabalhar com computação). Estava cursando Ciência da computação mas tranquei a matrícula e estou pretendendo migrar para Análise e Desenvolvimento de Sistemas.

Eu e minha noiva estamos planejando um intercâmbio para Dublin daqui uns anos. E foi meio que por acaso que descobri que lá tem um mercado fortíssimo pra área, e a intenção agora é me especializar o suficiente para quando irmos (quem sabe) não voltar mais.

Como eu sou iniciante na área, não manjo de muita coisa ainda, gostaria de saber se vocês podem me recomendar algo. Alguns cursos de especialização online, alguns livros, qualquer coisa sobre o assunto já me ajudaria bastante.

Desde já agradeço!

**Comment by luizbills** [comment]
E qual seu nivel de conhecimento hoje em dia?

**Comment by xJmsx** [comment]
Sei pouco, aprendi o básico de C++ na faculdade. Estou pensando em migrar para Python pois vi que é mais “receptivo” pra quem tá começando.

**Comment by luizbills** [comment]
É esquece o C++ por hora, C++ nem deve ter mercado para programador junior.
Python é uma boa escolha, porém se o seu objetivo é `frontend` então é melhor você considerar aprender javascript.
De qualquer forma, python vai te ajudar a pegar melhor os conceitos de programação.

Sugiro você fazer os cursos da codeacademy: https://www.codecademy.com/courses/learn-python
Ou procurar um livro bom de python.

Se quiser já começar pelo Javascript tem o livro [Eloquent Javascript](https://eloquentjavascript.net/)

**Comment by HmmerHead** [comment]
Sugiro a Alura pra você que está começando. Lá tem diversos cursos, e alguns vão do básico ao avançado. Sugiro bastante pra quem está começando, inclusive. Esse investimento vai valer a pena.

**Comment by xJmsx** [comment]
Eu esbarrei com esse site nesses últimos dias e vi que tem bastantes cursos na área mesmo, estava procurando saber se vale a pena mesmo. Vou dar uma olhada melhor nele mais tarde. Obrigado pelas dicas, pessoal!
